pages/clusters/queue_metrics.md

```diff
@@ -0,0 +1,24 @@
+# Queue metrics
+Queue metrics show you the most important statistics to help you optimize your agent setup and
+monitor your cluster's performance. These statistics are updated on the page every 10 seconds.
+
```

Comment:
Since this page is all about the panel in the UI, I think we should include a screenshot of it here. It'll also help ground the parts the rest of the page refer to.